src/btree/bt_sync.c

```diff
@@ -235,6 +252,7 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
 
         for (;;) {
             WT_ERR(__sync_dup_walk(session, walk, flags, &prev));
+        skip_dup_walk:
```

Comment:
Clang format is supposed to leave goto labels at the margin. I think there's a problem with ones that have underscores in them. And having it at the margin is important. Can you try renaming it to `skipwalk` and moving it to the margin and see if it stays there?

Reply:
corrected as per your suggestion and it works with clang-format.